examples/pyfluent-workflow/main.py

```diff
@@ -0,0 +1,220 @@
+# in this example. If you want working units for length to be other than inches
+# (for example, millimeters), make the appropriate change.
+
+fluent_session.tui.define.units("length", "in")
+
+
+# ## Enable heat transfer
+# Enable heat transfer by activating the energy equation.
+
+fluent_session.setup.models.energy.enabled = True
+
+# ## Create material
+# Create a material named ``"water-liquid"``.
+
+fluent_session.setup.materials.database.copy_by_name(type="fluid", name="water-liquid")
+
+# ## Set up cell zone conditions
+
+# Set up the cell zone conditions for the fluid zone (``elbow-fluid``). Set ``material``
+# to ``"water-liquid"``.
+
+fluent_session.setup.cell_zone_conditions.fluid["elbow-fluid"].material = "water-liquid"
+
+# ## Set up boundary conditions for CFD analysis
+
+# Set up the boundary conditions for the inlets, outlet, and walls for CFD
+# analysis.
+# - cold inlet (cold-inlet), Setting: Value:
+# - Velocity Specification Method: Magnitude, Normal to Boundary
+# - Velocity Magnitude: 0.4 [m/s]
+# - Specification Method: Intensity and Hydraulic Diameter
+# - Turbulent Intensity: 5 [%]
+# - Hydraulic Diameter: 4 [inch]
+# - Temperature: 293.15 [K]
+
+cold_inlet = fluent_session.setup.boundary_conditions.velocity_inlet["cold-inlet"]
+cold_inlet.get_state()
+cold_inlet.momentum.velocity.value = 0.4
+cold_inlet.turbulence.turbulent_specification = "Intensity and Hydraulic Diameter"
+cold_inlet.turbulence.turbulent_intensity = 0.05
+cold_inlet.turbulence. hydraulic_diameter = "4 [in]"
+cold_inlet.thermal.t.value = 293.15
+
+# - hot inlet (hot-inlet), Setting: Value:
+# - Velocity Specification Method: Magnitude, Normal to Boundary
+# - Velocity Magnitude: 1.2 [m/s]
+# - Specification Method: Intensity and Hydraulic Diameter
+# - Turbulent Intensity: 5 [%]
+# - Hydraulic Diameter: 1 [inch]
+# - Temperature: 313.15 [K]
+
+hot_inlet = fluent_session.setup.boundary_conditions.velocity_inlet["hot-inlet"]
+hot_inlet.momentum.velocity.value = 1.2
+hot_inlet.turbulence.turbulent_specification = "Intensity and Hydraulic Diameter"
+hot_inlet.turbulence.turbulent_intensity = 0.05
+hot_inlet.turbulence. hydraulic_diameter = "1 [in]"
+hot_inlet.thermal.t.value = 313.15
+
+# - pressure outlet (outlet), Setting: Value:
+# - Backflow Turbulent Intensity: 5 [%]
+# - Backflow Turbulent Viscosity Ratio: 4
+
+fluent_session.setup.boundary_conditions.pressure_outlet["outlet"].turbulence.turbulent_viscosity_ratio_real = 4
+
+# ## Initialize flow field
+
+fluent_session.solution.initialization.hybrid_initialize()
+
+# ## Solve for 150 iterations
+# Setting iteration count to 150 to solve the model.
+
+fluent_session.solution.run_calculation.iterate(iter_count=150)
+
+# ## Update Solution using Workbench Journal Commands
+
+wb.run_script_string("system1 = GetSystem(Name=\"FLU\")")
+wb.run_script_string("solutionComponent1 = system1.GetComponent(Name=\"Solution\")")
+wb.run_script_string("solutionComponent1.Update(AllDependencies=True)")
```

Comment:
You can do one call to run_script_string, and pass a multi-line string. It is more efficient that way.